x/tokenregistry/types/msgs.go

```diff
@@ -38,8 +38,8 @@ func (m *MsgRegister) ValidateBasic() error {
 	if err != nil {
 		return sdkerrors.Wrap(err, "invalid from address")
 	}
-	if m.Entry.Decimals <= 0 {
-		return errors.New("Decimals cannot be zero")
+	if m.Entry.Decimals < 0 {
```

Comment:
Are there any tokens that have 0 decimals? 
Might be a good idea to keep this in even if the code can handle 0 decimal tokens.